### ISSUE TITLE
Northeren Tracks: ReduceLROnPlateau

### ISSUE THREAD

**Issue by RasmusOrsoe** [graphnet-team/graphnet#473]
**What is benchmarked**
The piece-wise linear schedule does not always produce convergence on the northeren tracks dataset. Perhaps an appropriately parameterized ReduceLROnPlateau will do the trick?

**Target variables used for evaluation**
`'classification_emuon_entry'`

**Step-by-step**

1. Switch replace the piece-wise linear schedule with `ReduceLROnPlateau` (see snippet)
2. As an initial guess, set `learning_rate = 1e-3`,  `patience = 10` (early stopping) , `max_epochs = 200` and `patience = 4` (scheduler)
3. Run the training script
4. When the model converges, compare with current results using the plotting notebook.
5. Send plots to me on slack :-) 

```
scheduler_class = ReduceLROnPlateau
scheduler_kwargs = {'patience': 4}
scheduler_config = {'frequency': 1, 'monitor': 'val_loss'}

model = StandardModel(
        detector=detector,
        gnn=gnn,
        tasks=[task],
        optimizer_class=Adam,
        optimizer_kwargs={'lr': 1e-03, 'eps': 1e-03},
        scheduler_class= scheduler_class,
        scheduler_kwargs=scheduler_kwargs,
        scheduler_config=scheduler_config,
        coarsening = coarsening
     )
```

**Comment by jlkjlkj** [comment]
After some bugs were resolved it was easy to converge this and the GNN had some relative improvement in the high energies.

**Comment by jlkjlkj** [comment]
Issue can be closed.
